acceptancetests/jujupy/client.py

```diff
@@ -930,13 +934,16 @@ def update_user_name(self):
 
     def bootstrap(self, upload_tools=False, bootstrap_series=None,
                   credential=None, auto_upgrade=False, metadata_source=None,
-                  no_gui=False, agent_version=None):
+                  no_gui=False, agent_version=None, db_snap_path=None,
+                  db_snap_asserts_path=None):
         """Bootstrap a controller."""
         self._check_bootstrap()
         with self._bootstrap_config() as config_filename:
             args = self.get_bootstrap_args(
                 upload_tools, config_filename, bootstrap_series, credential,
-                auto_upgrade, metadata_source, no_gui, agent_version)
+                auto_upgrade, metadata_source, no_gui, agent_version,
+                db_snap_path=db_snap_path,
```

Comment:
if we're going to modify to start using kwargs then let's go that route and update the rest to be kwargs as well please.

Reply:
Make sense

Reply:
There was no need to have the last 2 as kwargs so I just made them regular args.